deploy/charts/rook-ceph/values.yaml

```diff
@@ -55,6 +55,10 @@ logLevel: INFO
 # -- If true, create & use RBAC resources
 rbacEnable: true
 
+rbacAggregate:
+  # -- If true, create a ClusterRole aggregated to [user facing roles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles)
+  enabled: false
```

Comment:
The roles are currently very specific to OBCs. Perhaps this setting should be more specific to OBCs, and if any other RBAC is aggregated in the future, they would have separate settings. What about this?
```
rbacAggregate:
  enableOBCs: true
```

Reply:
done